lovr-mouse.lua

```diff
@@ -107,11 +107,11 @@ end
 
 function mouse.newCursor(source, hotx, hoty)
   if type(source) == 'string' or tostring(source) == 'Blob' then
-    source = lovr.data.newTextureData(source, false)
+    source = lovr.data.newImage(source, false)
   else
     assert(tostring(source) == 'TextureData', 'Bad argument #1 to newCursor (TextureData expected)')
```

Comment:
And maybe TextureData should be renamed to Image data

Reply:
I agree, do you want to make that change here?

Reply:
Yep